src/arrayMethodPop.js

```diff
@@ -7,6 +7,17 @@
 function applyCustomPop() {
   [].__proto__.pop2 = function() {
     // write code here
+    return (this === []) ? this : this.splice(this.length - 1, 1)[0];
```

Comment:
How it pass tests?
1. pop should return removed element - not an array if an array is empty - should return undefined
2. I like the short variant, but splice is very powerful and it is too easy to write pop using splice
3. `[] !== []` [Object equality](https://medium.com/@sankalp.lakhina91/understanding-object-equality-in-javascript-ec3e7a2b5b1d)